### ISSUE TITLE
Tanks and temples dataset query

### ISSUE THREAD

**Issue by zubair-irshad** [jkulhanek/nerfbaselines#20]
Hi @jkulhanek, Great work! I have two questions regarding the tanks and temples dataset.

1. Is there an official train/val/test split you followed for baseline various methods on the tanks and temples dataset?

2. Did you run Colmap on it yourself or converted the raw poses provided [here](https://www.tanksandtemples.org/download/)?

Thanks!

**Comment by jkulhanek** [comment]
Hi @zubair-irshad.

1) I don't think there is an official split. I used the same train/test split strategy as in mipnerf360 (each 8-th image is testing). This is also the same split people use for training on a couple of TT scenes in papers like 3DGS, etc. See https://github.com/jkulhanek/nerfbaselines/blob/04f31bbcb96ef1f45f2db7c4b4115039b7e0bb94/nerfbaselines/datasets/tanksandtemples.py#L58
2) I ran COLMAP myself. The raw poses are only provided for train sets (5 scenes out of ~20) and the poses are less precise than running COLMAP from scratch.

Best,
Jonas
